### DISCUSSION TITLE
[issue] Relogin into Copilot

### DISCUSSION THREAD

**Discussion by Taisia-Sharapova** [community/community#8192] [Copilot Conversations]
Long story short, I have two GitHub accounts. One has Copilot access (this one), the other doesn't. I accidentally signed into the other account when setting up Copilot, and now it gives me the error "User not authorized," but doesn't allow to re-authorize anywhere. I've tried uninstalling the extension and then installing it back, but it didn't help. The error persists, and there's no prompt to log into GitHub.
How do I solve this?
Thanks in advance.

**Comment by Taisia-Sharapova** [comment]
Update: I installed the Nightly version and logged into Github successfully there, but it's not a long-term solution for sure. What if I want to use the mainline version?

**Comment by davecheney** [comment]
![Screen Shot 2021-11-29 at 11 41 55 am](https://user-images.githubusercontent.com/7171/143793206-cb03888d-7896-4575-8b04-6112f9bda97a.png)

What happens if you sign out?
